src/__tests__/renderIntoDocument.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import {renderIntoDocument, clearDocument} from '../'
+
+afterEach(clearDocument)
+
+it('renders button into document', () => {
+  const ref = React.createRef()
+  renderIntoDocument(<div id="test" ref={ref} />)
+  expect(document.body.querySelector('#test')).toBe(ref.current)
```

Comment:
Could remove the `id` and the assertion could be: `expect(document.body.firstChild).toBe(ref.current)`